app/models/user.js

```diff
@@ -36,12 +36,16 @@ module.exports = (sequelize, DataTypes) => {
     // associations can be defined here
   };
 
-  User.createModel = user => {
+  User.createModel = (user, next = null) => {
```

Comment:
Why would the model know about the next middleware? Its not in the model scope to know about that kind of stuff. You should never let the next middleware go anywhere outside the controller or some other middlewares.